waves/src/App.test.tsx

```diff
@@ -150,25 +150,34 @@ test("Swap asset types", () => {
         beta: AssetType.USDT,
     };
 
-    let newValue = AssetType.BTC;
     expect(
         reducer(initialState, {
             type: "SwapAssetTypes",
+            value: {
+                betaAmount: initialState.alpha.amount,
+            },
         }).alpha.type,
     ).toBe(initialState.beta);
 
     expect(
         reducer(initialState, {
             type: "SwapAssetTypes",
+            value: {
+                betaAmount: initialState.alpha.amount,
+            },
         }).beta,
     ).toBe(initialState.alpha.type);
 
     // amounts should be flipped as well
+    let betaAmount = calculateBetaAmount(initialState.alpha.type, initialState.alpha.amount, initialState.rate);
```

Comment:
This still accesses the `rate` from the state but in the actual code, the state no longer has the rate.